### PR TITLE
WIP Update Sample App

### PR DESCRIPTION
**Pull request by flschweiger**
Currently a WIP, come back soon 🚧

### CODE REVIEW

File: example/android/app/build.gradle [hunk 1]
```diff
@@ -38,9 +38,9 @@ android {
 
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
-        applicationId "io.wiredash.example"
+        applicationId "io.wiredash.flutter_app"
```
Comment (flschweiger) [inline example/android/app/build.gradle:41]: Any reason why changing the applicationId to `.flutter_app`?

**Review by flschweiger — COMMENTED** [review]
LGTM, I have changed a few things in the actual sample app but otherwise great job 👍 Just let me know if we should keep the new package name or if you want to adjust it back to `example` @atoka93

**Comment by passsy** [comment]
How to recreate/update the example app:

```bash
flutter channel beta
flutter upgrade
flutter config --enable-web    
flutter config --enable-macos-desktop
rm -rf example
flutter create --org io.wiredash --project-name example example
```

File: example/android/app/build.gradle (hunk 1 above)
Comment (atoka93) [reply example/android/app/build.gradle:41]: Unintended, will revert it.

File: example/lib/main.dart [hunk 2]
```diff
@@ -100,23 +100,11 @@ class _HomePage extends StatelessWidget {
...
-            },
-            child: Card(
-              child: ListTile(
-                title: Text('ListTile #$index'),
-              ),
-            ),
+          return ListTile(
+            title: Text('Sample Item #$index'),
+            subtitle: const Text('Tap me to open a new page'),
+            onTap: () => _openDetailsPage(context, index),
```
Comment (atoka93) [inline example/lib/main.dart:107]: Totally forgot about that. 💪

File: example/.metadata [hunk 3]
```diff
@@ -4,7 +4,7 @@
 # This file should be version controlled and should not be manually edited.
 
 version:
-  revision: 659dc8129d4edb9166e9a0d600439d135740933f
-  channel: beta
+  revision: 022b333a089afb81c471ec43d1f1f4f26305d876
+  channel: dev
```
Comment (passsy) [inline example/.metadata:8]: Was the upgrade to `dev` intentional?

Comment (flschweiger) [reply example/.metadata:8]: It was for creating the MacOS project files but we can also revert to beta.

Comment (passsy) [reply example/.metadata]: This did now work for me. I wasn't able to compile the app at all on iOS. I reverted back to stable (to recreate the project). It now works with Flutter stable/beta/dev

Comment (flschweiger) [reply example/.metadata]: Does this affect MacOS?

Comment (passsy) [reply example/.metadata]: Works like a charm (on dev `1.24.0-10.2.pre`)

**Review by passsy — APPROVED** [review]

**Pull request merged**